bom/application/pom.xml

```diff
@@ -433,26 +432,20 @@
                 <version>${opensearch-testcontainers.version}</version>
             </dependency>
             <!-- OpenTelemetry components, imported as a BOM -->
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom</artifactId>
-                <version>${opentelemetry.version}</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
-            <dependency>
-                <groupId>io.opentelemetry</groupId>
-                <artifactId>opentelemetry-bom-alpha</artifactId>
-                <version>${opentelemetry.version}-alpha</version>
-                <type>pom</type>
-                <scope>import</scope>
-            </dependency>
+            <!-- opentelemetry-instrumentation-bom-alpha includes the main and alpha-bom inside.-->
+            <!-- Also includes stable semantic conventions-->
             <dependency>
                 <groupId>io.opentelemetry.instrumentation</groupId>
                 <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
-                <version>${opentelemetry-alpha.version}</version>
+                <version>${opentelemetry-instrumentation.version}</version>
                 <type>pom</type>
                 <scope>import</scope>
+                <exclusions>
+                    <exclusion>
+                        <groupId>io.opentelemetry.javaagent</groupId>
+                        <artifactId>*</artifactId> <!-- Exclude all artifacts in this group -->
+                    </exclusion>
+                </exclusions>
```

Comment:
@brunobat I think it's a good step in the right direction.

Some additional data:

```
ack '<groupId>' opentelemetry-instrumentation-bom-alpha-2.10.0-alpha.pom | wc -l
363

ack '<groupId>' opentelemetry-instrumentation-bom-alpha-2.10.0-alpha.pom | sed 's/ //g' | sort | uniq -c | sort -n
      2 <groupId>io.opentelemetry</groupId>
      2 <groupId>io.opentelemetry.semconv</groupId>
      9 <groupId>io.opentelemetry.javaagent</groupId>
     78 <groupId>io.opentelemetry.instrumentation</groupId>
    272 <groupId>io.opentelemetry.javaagent.instrumentation</groupId>
```

Reply:
I don't think we need `io.opentelemetry.javaagent.instrumentation` either. Will run a build here.

Reply:
Yeah in my issue, I was talking about anything under `io.opentelemetry.javaagent`. I should have been more precise.